lib/live_cluster/manage_controller.py

```diff
@@ -365,35 +365,26 @@ async def _do_default(self, line):
 
 
 @CommandHelp(
-    "Change your password",
+    "Change your own password",
     modifiers=(
-        ModifierHelp("username", "User that needs a new password"),
         ModifierHelp(
             "old",
-            "Current password for the user. User will be prompted if no password is provided",
+            "Current password. User will be prompted if no password is provided",
         ),
         ModifierHelp(
             "new",
-            "New password for the user. User will be prompted if no password is provided",
+            "New password. User will be prompted if no password is provided",
         ),
     ),
-    usage="<username> [old <old-password>] [new <new-password>]",
+    usage="[old <old-password>] [new <new-password>]",
 )
 class ManageACLChangePasswordUserController(ManageLeafCommandController):
     def __init__(self):
         self.modifiers = set(["old", "new"])
-        self.required_modifiers = set(["user"])
+        self.required_modifiers = set([])
         self.controller_map = {}
 
     async def _do_default(self, line):
-        username = util.get_arg_and_delete_from_mods(
-            line=line,
-            arg="user",
-            return_type=str,
-            default="",
-            modifiers=self.required_modifiers,
-            mods=self.mods,
-        )
```

Comment:
This is a breaking change, we should increment the version number and document it as such. Consider, you could continue to parse the user argument to be backwards compatible.